src/pages/Overview.page.js

```diff
@@ -0,0 +1,33 @@
+const { BaseSwagLabPage } = require('./BaseSwagLab.page');
```

Comment:
You use **import** in some modules, in another **require** for consistency better to use one of it. Or maybe that was a tack to use different ways\syntaxisi of the impot?